### PR TITLE
Merging changes for UserId integration

### PR DESCRIPTION
**Pull request by ChristopherGarsia**

### CODE REVIEW

File: src/Profile/Profile.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Profile extends Component {
   constructor() {
     super();
     this.state = {
-      account: 'test@email',
+      userId: 'suck',
```
Comment (VillanuevaK) [inline src/Profile/Profile.js]: Um

Comment (priya107) [reply src/Profile/Profile.js]: Uh

Comment (oshijoshi) [reply src/Profile/Profile.js]: Oh

File: src/Home.js [hunk 2]
```diff
@@ -33,15 +33,16 @@ class Home extends Component {
...
       let newState = [];
       for(let item in items) {
         newState.push({
           id:item,
-          title:items[item].name,
-          user:items[item].bio,
-          venmo:items[item].venmo
+          title:items[item].Name,
+          user:items[item].Bio,
+          venmo:items[item].Venmo
```
Comment (oshijoshi) [inline src/Home.js]: Miss the lower case aesthetic...but i guess it's fine

File: src/Profile/Profile.js (hunk 1 above)
Comment (Ericg21) [reply src/Profile/Profile.js]: Que?

File: src/Home.js (hunk 2 above)
Comment (ChristopherGarsia) [reply src/Home.js]: lower case is so 2018...

**Pull request merged**